packages/core/botpress/src/web/components/Util/BackendToast.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import { toast } from 'react-toastify'
+import _ from 'lodash'
+
+import EventBus from '~/util/EventBus'
+
+export default class BackendToast extends Component {
+  constructor(props) {
+    super(props)
+
+    const uniqueCodes = _.uniq(props.codes)
+    uniqueCodes.forEach(code => {
+      let id = null
```

Comment:
would make sense to call it `mostRecentToastId`?

Reply:
Can be